wai-aria/role/role_none_conflict_resolution.tentative.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Role None Conflict Resolution Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    
+    <!-- no conflict, since not focusable; inner button should be exposed as generic - children presentational - //https://github.com/web-platform-tests/interop-accessibility/issues/161 -->
+    <h2>ARIA role="button" nesting ARIA role="button" (no presentational roles conflict resolution, since not focusable)</h2>
+    <div role="button">
```

Comment:
missing `data-expectedrole`, `class="ex-role"`, `data-testname`, etc.

Reply:
Hi @cookiecrook, I didn't set any attributes because the goal was to test only the nested button, not the wrapper. However, if we'd like to test the wrapper as well, I'm more than happy to add the necessary attributes.

Reply:
I added additional examples to ensure we retain the scenario where presentational roles conflict resolution isn't triggered.